lib/rubocop-rspec.rb

```diff
@@ -9,21 +9,23 @@
 require_relative 'rubocop/rspec/version'
 require_relative 'rubocop/rspec/inject'
 require_relative 'rubocop/rspec/node'
-require_relative 'rubocop/rspec/top_level_describe'
 require_relative 'rubocop/rspec/wording'
 require_relative 'rubocop/rspec/language'
 require_relative 'rubocop/rspec/language/node_pattern'
-require_relative 'rubocop/rspec/top_level_group'
+
+require_relative 'rubocop/cop/rspec/mixin/top_level_describe'
```

Comment:
When we change the folder like this, shouldn’t we also change the namespace? E.g. `RuboCop::RSpec::TopLevelDescribe` -> `RuboCop::Cop::RSpec::TopLevelDescribe`.

Reply:
Agree.

@mockdeep We've just renamed `RuboCop::Cop::RSpec::Cop` to `RuboCop::Cop::RSpec::Base` following RuboCop core's naming.
If we happen to move classes/modules around namespaces, and if it happens those are used by custom cops, we'll break them with a minor version update.
So for `Base`, we've decided to keep the old class, `Cop` as well until 2.0.
I don't see how it would be possible to release this change with namespace before we release 2.0 without similar breakage risks.

AFAI understand #863 depends on this pull request. But if this is going to be released in 2.0, that means that #863 will only be released as `enabled` in 3.0, until then it will be `pending`.

So I propose two options for #863:
 - untangle it from this pull request, and release separately
 - keep it dependent on this pull request and release post-2.0

Please let me know what you think.